cdap-proto/src/main/java/co/cask/cdap/proto/NamespaceMeta.java

```diff
@@ -28,10 +28,17 @@
 public final class NamespaceMeta {
 
   public static final NamespaceMeta DEFAULT =
-    new NamespaceMeta.Builder().setName(NamespaceId.DEFAULT).setDescription("Default Namespace").build();
+    new NamespaceMeta.Builder()
+      .setName(NamespaceId.DEFAULT)
+      .setDescription("The default namespace, which is always available in CDAP, and can be used in " +
```

Comment:
Can we remove the product name from description.

Reply:
fixed